### ISSUE TITLE
PhantomJsDriverManager exception

### ISSUE THREAD

**Issue by rosolko** [bonigarcia/webdrivermanager#20]
Seems like something broken in this manager.

Manager unpack all files from zip archive and didn't clean output folder.
![image](https://cloud.githubusercontent.com/assets/11458319/13701935/2e91629a-e79c-11e5-857b-9754ccb4085b.png)

Also method `extract` from `Downloader.class` return `third-party.txt` file as binary file.
![image](https://cloud.githubusercontent.com/assets/11458319/13701962/5a5dac76-e79c-11e5-87e6-a62c8c1f1b23.png)

In accordance to this behavior tests throw exception:
``` java 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
Mar 11, 2016 3:17:46 PM org.openqa.selenium.phantomjs.PhantomJSDriverService <init>
INFO: executable: C:\Users\Aliaksandr_Rasolka\IdeaProjects\sandbox\bin\phantomjs\2.1.1\phantomjs-2.1.1-windows\third-party.txt
Mar 11, 2016 3:17:46 PM org.openqa.selenium.phantomjs.PhantomJSDriverService <init>
INFO: port: 33403
Mar 11, 2016 3:17:46 PM org.openqa.selenium.phantomjs.PhantomJSDriverService <init>
INFO: arguments: [--webdriver=33403, --webdriver-logfile=C:\Users\Aliaksandr_Rasolka\IdeaProjects\sandbox\phantomjsdriver.log]
Mar 11, 2016 3:17:46 PM org.openqa.selenium.phantomjs.PhantomJSDriverService <init>
INFO: environment: {}
Mar 11, 2016 3:18:06 PM org.openqa.selenium.os.UnixProcess checkForError
SEVERE: org.apache.commons.exec.ExecuteException: Execution failed (Exit value: -559038737. Caused by java.io.IOException: Cannot run program "C:\Users\Aliaksandr_Rasolka\IdeaProjects\sandbox\bin\phantomjs\2.1.1\phantomjs-2.1.1-windows\third-party.txt" (in directory "."): CreateProcess error=193, %1 is not a valid Win32 application)
```

This exception throw only on first run (when you didn't have any phantomjs files in output folder), I think it's because in the method all of the files marked as executable.

**Comment by bonigarcia** [comment]
Solved in version 1.4.1.

**Comment by djangofan** [comment]
yeah, I use PhantomJS  to run unit tests in my automation framework.  I haven't had a problem with PhantomJS & WebDriverManager  on version 1.4.2 or 1.4.3 .
